app/src/main/java/org/wildaid/ofish/data/RealmDataSource.kt

```diff
@@ -189,11 +197,15 @@ class RealmDataSource(context: Context) {
             .toList()
     }
 
-    fun getAmountOfDrafts(): Int {
-        return realm.where<Report>()
-            .equalTo(DRAFT, true)
-            .findAll()
-            .count()
+    fun getAmountOfDraftsForCurrentOfficer(): Int {
+        val currentOfficer = getCurrentOfficer()
```

Comment:
Data source should not know about officer. Repository should use officer as argument to data source